src/duckling/game/collision/collision-attribute.ts

```diff
@@ -1,26 +1,13 @@
 import {Attribute, Entity} from '../../entitysystem/entity';
 import {Vector} from '../../math/vector';
 import {Box2} from '../../math/box2';
+import {SelectOption, toSelectOptions} from '../../controls';
 
 export const COLLISION_KEY = "collision";
 
-/**
- * Describes how the entity behaves in the solid body system.
- */
-export enum BodyType {
-    None,
-    Solid,
-    Environment
-}
+export var BodyTypeSelect : SelectOption[] = toSelectOptions(["none", "solid", "environment"]);
```

Comment:
You can make this type safe:
toSelectOptions<T>(keys : T[]) {
	...	
}

type BodyType = "none" | "solid" | "environment";
var BodyTypeOptions = toSelectOptions<BodyType>(["none", "solid", "environment"])